client/src/components/AddRecordButton.tsx

```
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  ButtonWithDropdown,
+  PlusCircle,
+  TextWithIcon,
+} from '@ctoec/component-library';
+import { Organization } from '../shared/models';
+import { Link, useHistory } from 'react-router-dom';
+
+// TODO: change after org => providers PR is merged
+export const AddRecordButton = ({
+  providers,
+}: {
+  providers?: Organization[];
+}) => {
+  const history = useHistory();
+
+  if (!providers) {
+    return <></>;
```

we should implement and error boundary before launch and throw errors consistently instead of returning empty fragments (i am the biggest culprit here!!! but just saying this "out loud" so i remember to get it scoped)

---

It exists in component library so we really just have to start using it!